### ISSUE TITLE
Lost header words 2.1.13

### ISSUE THREAD

**Issue by dfgroff** [geoffhumphrey/brewcompetitiononlineentry#970]
Version:2.1.13

Installation URL:http://thoroughbrews.org/kbotb1

Is your installation hosted on brewcompetition.com or brewcomp.com?NO

Description of Issue: Did the upgrade now my site doesn't look correct. I lost the words in the header and when I can make it to other pages I don't have any of the fields anymore. 

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by geoffhumphrey** [comment]
This has to do with the installation not "finding" the correct language file (which, currently, is only US English) because it hasn't been set in the session. 

To correct this, make sure that US English is selected as your language on Admin > Preferences > Site Preferences. You may need to clear your browser cache to see the words again.

If that doesn't work, access your database directly (via phpMyAdmin, shell access, etc.) and change the value in the prefsLanguage column to _en-US_ (see example below in phpMyAdmin).
![2018-09-11_11-45-36](https://user-images.githubusercontent.com/5216502/45377631-605b3480-b5b8-11e8-8abf-324df0f95b2b.jpg)

Then, clear your cache to end the PHP session.

Hope that helps!

**Comment by dfgroff** [comment]
Thank you that fixed it. I really appreciate the help.

**Comment by dfgroff** [comment]
If I make a change on the site now I'm getting Unknown column 'archiveScoresheet' in 'field list' when I hit save. Any ideas on what this could be? Thanks

**Comment by djmitche** [comment]
It looks like this field was created as a rename of `archiveBrewingTableName` in a25aea3b3dd6e3aee854e23078ef619b02f6bef2.  Is there a corresponding DB migration?

**Comment by geoffhumphrey** [comment]
Yes, it was implemented with version 2.1.12's updating scripts - specifically around line 760 in /update/off_schedule_update.php - with this code:

```
if (!check_update("archiveScoresheet", $prefix."archive")) {

	$updateSQL = sprintf("ALTER TABLE `%s` CHANGE `archiveBrewingTableName` `archiveScoresheet` CHAR(1) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NULL DEFAULT NULL;",$prefix."archive");
	mysqli_select_db($connection,$database);
	mysqli_real_escape_string($connection,$updateSQL);
	$result = mysqli_query($connection,$updateSQL);

	// Set all to 'J'
	$updateSQL = sprintf("UPDATE `%s` SET archiveScoresheet='J';",$prefix."archive");
	mysqli_select_db($connection,$database);
	mysqli_real_escape_string($connection,$updateSQL);
	$result = mysqli_query($connection,$updateSQL);

}
```

You can force an update by changing the FORCE_UPDATE constant in the paths.php script to TRUE, uploading to your installation, and accessing index.php a single time. This should make the necessary DB changes.

After that, change the var back to FALSE and re-upload.

**Comment by dfgroff** [comment]
Thanks that fixed it.
